src/parser/test/ParserTest.cpp

```diff
@@ -3015,4 +3015,51 @@ TEST_F(ParserTest, ShowAndKillQueryTest) {
     ASSERT_EQ(result.value()->toString(), "KILL QUERY (session=123, plan=123)");
   }
 }
```

Comment:
Add test cases for the new syntax.

Reply:
For the time reason, I will add it in next pr